<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralWindow</class>
 <widget class="QMainWindow" name="NeuralWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Stream</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="trace_stream_combobox"/>
      </item>
      <item>
       <widget class="QPushButton" name="import_recording">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_recording">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Sort/Cluster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="full_trace_push_button">
        <property name="text">
         <string>Full trace</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="space_spinbox">
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
        <item>
         <widget class="TraceAxis" name="trace_view"/>
        </item>
        <item>
         <widget class="TraceAxis" name="avg_trace_view"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="behavior_groupbox">
        <property name="title">
         <string>Behaviors</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QScrollArea" name="behavior_scrollarea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>105</width>
              <height>510</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionImportNeuralRecord">
   <property name="text">
    <string>Import neural recording</string>
   </property>
  </action>
  <action name="actionResetNeuralRecording">
   <property name="text">
    <string>Clear neural recording</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TraceAxis</class>
   <extends>QGraphicsView</extends>
   <header>bannotator.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
